app/models/filtering_test.rb

```diff
@@ -43,7 +43,11 @@ def pick_filter_criteria
     return unless options && options['filters']
     # select a random patient
     prng = Random.new(rand_seed.to_i)
-    rand_record = records.sample(random: prng)
+    random_mrn = ''
+    while random_mrn[8] != '-'
```

Comment:
Is it worth it to make this `8` index be turned into a constant somewhere? Good coding practice would dictate that, at least.